recipes/hurry.filesize/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set version = '0.9' %}
+{% set sha256 = 'f5368329adbef86accd3bc9490522340bb79260455ae89b1a42c10f63801b9a6' %}
+
+package:
+  name: hurry.filesize
+  version: {{ version }}
+
+source:
+  url: https://pypi.python.org/packages/source/h/hurry.filesize/hurry.filesize-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install --no-deps --ignore-installed . -vv
+
+requirements:
+  host:
+    - python
+    - pip
+
+  run:
+    - python
+
+test:
+  imports:
+    - hurry
+    - hurry.filesize
+
+about:
+  home: https://pypi.org/project/hurry.filesize/
+  license: ZPL 2.1
```

Comment:
This is going to need a `license_file`. I'm not sure how best to handle this as one isn't distributed with the code or appears in an upstream repo (since there doesn't appear to be one). It may be best just to copy one from online:

https://directory.fsf.org/wiki/License:ZPL-2.1

although I'm not sure about the mention of a copyright holder notice.

Reply:
Thanks for review. What is the recommended way to include this file -- through a patch?

Reply:
https://conda-forge.org/docs/maintainer/adding_pkgs.html#packaging-the-license-manually

Reply:
Thanks! Pushed fix.